src/functions/__tests__/fast-login.test.ts

```diff
@@ -344,4 +391,149 @@ describe("Fast Login handler", () => {
       );
     }
   );
+
+  it(`GIVEN a valid LolliPoP request 
+      WHEN the nonce is missing 
+      THEN an unauthorized error is returned`, async () => {
+    const req: H.HttpRequest = {
+      ...H.request("https://api.test.it/"),
+      headers: {
+        ...validLollipopHeaders,
+        ...validFastLoginAdditionalHeaders,
+        ["signature-input"]: `sig1=("x-pagopa-lollipop-original-method" "x-pagopa-lollipop-original-url");created=1698315748;alg="ecdsa-p256-sha256";keyid="iwBFlFaCWaLnrCckGIyWMJBnfDkEJ-mgxZVzGICmkwU"`
+      }
+    };
+    const result = await makeFastLoginHandler({
+      ...httpHandlerInputMocks,
+      input: req,
+      fnLollipopClient: mockedFnLollipopClient,
+      blobService: mockBlobService,
+      redisClientTask: mockRedisClientTask
+    })();
+    expect(mockValidateSignature).toHaveBeenCalled();
+    expect(mockDel).not.toHaveBeenCalled();
+    expect(getAssertionMock).not.toHaveBeenCalled();
+    expect(mockUpsertBlobFromObject).not.toHaveBeenCalled();
+    expect(mockUpsertBlobFromObject).not.toHaveBeenCalled();
+    expect(result).toEqual(
+      E.right(
+        expect.objectContaining({
+          statusCode: 401,
+          body: expect.objectContaining({
+            title:
+              "Could not delete nonce: [Could not retrieve nonce from signature-input.]",
+            status: 401
+          })
+        })
+      )
+    );
+  });
+
+  it(`GIVEN a valid LolliPoP request 
```

Comment:
ditto

Reply:
added a comment in https://github.com/pagopa/io-functions-fast-login/pull/27/commits/24017f00ff4d81f3fd02f3eba39eac89e1b43890